envoy-filters/mgw-source/filters/http/mgw-wasm-websocket/filter.cc

```diff
@@ -335,5 +372,19 @@ void MgwWebSocketContext::establishNewStream() {
 // Callback used by the handler to update throttle period.
 void MgwWebSocketContext::updateThrottlePeriod(const int throttle_period){
   this->throttle_period_ = throttle_period;
-  LOG_TRACE("Throttle period updated to"+ std::to_string(throttle_period));
+  LOG_TRACE("Throttle period updated to"+ std::to_string(throttle_period) + std::string(" : ") + this->x_request_id_);
+}
+
+void MgwWebSocketContext::sendEnforcerRequest(MgwWebSocketContext* websocContext, WebSocketFrameRequest request) {
+  if (websocContext->handler_state_ == HandlerState::OK) {
+    LOG_TRACE(std::string("gRPC bidi stream available. publishing frame data...") + std::string(" : ") + this->x_request_id_);
+  } else {
+    establishNewStream();
+  }
+  auto ack = websocContext->stream_handler_->send(request, false);
+  if (ack != WasmResult::Ok) {
+    LOG_WARN(std::string("error sending frame data")+ toString(ack) + std::string(" : ") + this->x_request_id_);
```

Comment:
when you are testing let's look for this log too.

Reply:
Ack